test/libyul/EVMCodeTransformTest.cpp

```diff
@@ -46,7 +46,6 @@ EVMCodeTransformTest::EVMCodeTransformTest(std::string const& _filename):
 	m_source = m_reader.source();
 	m_stackOpt = m_reader.boolSetting("stackOptimization", false);
 	m_expectation = m_reader.simpleExpectations();
-	m_shouldRun = CommonOptions::get().evmDialect().evmVersion() == EVMVersion{};
```

Comment:
You should also pass `CommonOptions::get().evmVersion()` to `YulStack` below. Otherwise the test sill runs with hard-coded version.

Also, we should do the same for `EOFVersion`.

Reply:
Yeah, fair!